src/views/LoginView.jsx

```diff
@@ -1,10 +1,63 @@
-import React from 'react';
-import Login from '../components/Login';
+import React, { Component } from "react";
+import Login from "../components/Login";
+import { connect } from "react-redux";
+import loginFetch from "../store/actions/async/login";
+import { removeLoginError } from "../store/actions/sync/login";
 
-const LoginView = () => (
-    <div>
-        <Login />
-    </div>
-);
+export class LoginView extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+        email: "",
+        password: ""
+        };
+    }
 
-export default LoginView;
+    onSubmitHandler = e => {
+        e.preventDefault();
+        const { email, password } = this.state;
+        const data = { email, password };
+        const url =
+        "https://ah-backend-summer-staging.herokuapp.com/api/v1/users/login";
+        const payload = {
+        data,
+        history: this.props.history,
+        url
+        };
+        this.props.login(payload);
+    };
+
+    onChangeHandler = e => this.setState({ [e.target.name]: e.target.value });
+
+    onDismissHandler = () => this.props.dismissLoginError();
+
+    render() {
+        const loginProps = {
+        email: this.state.email,
+        password: this.state.password,
+        onDismissHandler: this.onDismissHandler,
+        loginError: this.props.loginError,
+        onSubmitHandler: this.onSubmitHandler,
+        onChangeHandler: this.onChangeHandler
```

Comment:
Nice work here @oma0256 and @missvicki. The ui looks good too. However, aren't we missing the declaration of proptypes in this file?

Reply:
@pbkabali currently the LoginView component does not take in any props

Reply:
@pbkabali these props are being passed to the Login component please check the login.jsx for their declaration as they were declared their.

Reply:
True, but I still believe they need to be added to the validation. Is hound not complaining about this?

Reply:
@karuhanga and @pbkabali hound did complain and the fix was added in login.jsx. after which we didn't have issues with declaring proptypes. I think it would be a repeatition if we declared them here as well